website/archiver/utils.py

```diff
@@ -25,6 +26,7 @@ def send_archiver_size_exceeded_mails(src, user, stat_result):
         mail=mails.ARCHIVE_SIZE_EXCEEDED_USER,
         user=user,
         src=src,
+        can_change_preferences=False,
```

Comment:
At some point extra text like "You can change your email preferences... " was added to the nofify_base email template. This text is not relevant to any archiver emails.